src/public/sass/ITCSS/06_components/pages/product/product-info/_product-info.scss

```
@@ -0,0 +1,115 @@
+@use 'sass:color';
+@use '../../../../01_settings/global' as *;
+@use '../../../../01_settings/colors' as *;
+@use '../../../../02_tools/breakpoints' as *;
+@use '../../../../02_tools/mixins' as *;
+
+.product-info {
+  width: 90%;
+  max-width: 75rem;
+  margin: 1.25rem auto;
+  scroll-margin-top: 8.125rem;
+
+  &__heading {
+    margin: 0 0 0.625rem 0;
+    text-align: left;
+    text-transform: capitalize;
+    transition: color 1000ms ease-in-out;
+    @include fluid-text(1.8rem, 5rem);
+
+    &--description {
+      color: $color-sky-blue;
+    }
+
+    &--tech-info {
+      color: $color-royal-blue;
+    }
+
+    &--download-area {
+      color: $color-brand-dark;
+    }
+  }
+
+  &__text-container {
+    padding-top: 4rem;
+    transition: border-color 1000ms ease-in-out;
+
+    &--description {
+      padding-top: 0;
+      border-top: 0.8rem solid $color-teal-blue;
+    }
+
+    &--tech-info {
+      padding-top: 0;
+      border-top: 0.8rem solid $color-bright-blue;
+    }
+
+    &--download-area {
+      padding-top: 0;
+      border-top: 0.8rem solid $color-sky-blue;
+    }
+
+    @include tablet() {
+      padding-top: 0;
+      padding-left: 3rem;
+      border-top: none;
+      margin-left: 0.2rem;
+
+      &--description {
+        padding-top: 0;
+        border-left: 0.8rem solid $color-teal-blue;
+      }
+
+      &--tech-info {
+        padding-top: 0;
+        border-left: 0.8rem solid $color-bright-blue;
+      }
+
+      &--download-area {
+        padding-top: 0;
+        border-left: 0.8rem solid $color-sky-blue;
+      }
+    }
+  }
+
+  &__text {
+    margin: 0.9375rem 0 -0.625rem 0;
+    color: color.adjust($color-gray-60, $lightness: 5%);
+    font-size: 1.25rem;
+    line-height: 1.4;
+
+    &--title {
+      margin-right: 0.5rem;
+      font-weight: 700;
```

```suggestion
      font-weight: $weight-bold;
```
remember to change numbers on font-weight for variables and try to not create colors on component files, and check it on all files of product page